variables.tf

```
@@ -52,6 +52,12 @@ variable "enable_pod_security_policy" {
   default     = false
 }
 
+variable "enable_http_application_routing" {
```

```suggestion
variable "http_application_routing_enabled" {
```
We have chosen in our development guidelines (not public) to use the attribute instead of the verb. Despite we still have work to do, this is the mainly used implementation.